playwright.config.js

```diff
@@ -0,0 +1,37 @@
+const { defineConfig, devices } = require('@playwright/test');
+const dotenv = require('dotenv');
+
+
+// Read from default ".env" file.
```

Comment:
Is there a .env-example file? Could be good to add one if not.

Reply:
There is not with this yet, plans were to add it in one of the following couple of updates.